tooling/nargo_cli/src/cli/test_cmd.rs

```diff
@@ -275,32 +283,39 @@ fn display_test_report(
 
     write!(writer, "[{}] ", package.name).expect("Failed to write to stderr");
 
-    let count_all = test_report.len();
     let count_failed = test_report.iter().filter(|(_, status)| status.failed()).count();
-    let plural = if count_all == 1 { "" } else { "s" };
-    if count_failed == 0 {
+    let count_passed = test_report.iter().filter(|(_, status)| status.passed()).count();
+    let count_skipped = test_report.iter().filter(|(_, status)| status.skipped()).count();
+    let plural_failed = if count_failed == 1 { "" } else { "s" };
+    let plural_passed = if count_passed == 1 { "" } else { "s" };
+    let plural_skipped = if count_skipped == 1 { "" } else { "s" };
+    let mut previous = false;
+    if count_passed > 0 {
```

Comment:
```suggestion
    if count_passed > 0 || count_skipped == 0 && count_failed == 0 {
```
Previously if there were 0 tests executed it still gave feedback that "0 tests passed" as a confirmation that something happened. Maybe not the best message, but just wanted to highlight that it was there.

Reply:
Arguably Rust's version is unambiguous:
```
running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 3 filtered out; finished in 0.00s
```

Reply:
Actually, never mind, I see that the number of filtered tests is displayed before and after, so not displaying "0 tests passed" will make even more sense 👍 

```console
❯ cargo run -q -p nargo_cli -- --program-dir . test blah

[keccak256] Running 0 test functions
[keccak256] 0 tests passed
Found 0 tests containing 'blah'.
```